allure-commandline/src/main/scripts/allure

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ -z "$ALLURE_HOME" ]; then
+
+    SOURCE="${BASH_SOURCE[0]}"
```

Comment:
man there is nothing special. If I don't know php doesn't mean that ppl should comment each line of their php code imho